style.css

```diff
@@ -0,0 +1,19 @@
+#welcome-section{
+  color: blue;
+  background-color:crimson;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-weight: 600;
+}
+
```

Comment:
unwanted space.

Reply:
- [ ] Kindly consider phrasing your comments politely and also providing more context to explain to the student why they should remove unwanted spaces from their codebase. Thanks 🙏😊 to you the student will learn from their mistakes and will not repeat them in their future projects.